setup.py

```diff
@@ -6,6 +6,10 @@
 
 from setuptools import setup
 
+from pelican import get_version
```

Comment:
This is an issue. Importing `pelican` will try to import loads of not-yet-installed dependencies.

pip 19+ is fine because it added `pyproject.toml` but earlier versions will fail. Similarly `python setup.py install` will also fail.

Reply:
I wondered about that. Thanks for pointing that out. At the minor expense of making it less DRY, I re-pushed with a slimmer version of the `get_version()` function added directly to `setup.py`. Do you see any issues with how things look now? Anything you think I should change?

Reply:
Is the `get_version` in `__init__.py` necessary? Only way it would run successfully is to import pelican from source folder (not installed) but all the dependencies were installed. Odd condition.

Reply:
No, I don't think the `get_version()` function is necessary. I removed it in the latest commit and moved its basic logic to the `except` block below. Does that make things better?

Reply:
I understand better now your point about it being an odd condition that doesn't need to be addressed. I removed that logic from `__init__.py`.